lib/web_ui/lib/src/engine/dom.dart

```diff
@@ -219,6 +219,18 @@ extension DomElementExtension on DomElement {
   external void remove();
   external void setAttribute(String name, Object value);
   void appendText(String text) => append(createDomText(text));
+  external void removeAttribute(String name);
+  external set tabIndex(int? value);
+  external int? get tabIndex;
+  external void focus();
+  int get scrollTop =>
+      js_util.getProperty(this, 'scrollTop').round();
+  set scrollTop(int value) =>
+      js_util.setProperty<int>(this, 'scrollTop', value.round());
+  int get scrollLeft =>
+      js_util.getProperty(this, 'scrollLeft').round();
+  set scrollLeft(int value) =>
```

Comment:
Same here

Reply:
See earlier comment, I'm not sure this is guaranteed to be an int in all cases.